konflux-ci/build-service/core/kustomization.yaml

```diff
@@ -12,7 +12,7 @@ namespace: build-service
 images:
   - name: quay.io/konflux-ci/build-service
     newName: quay.io/konflux-ci/build-service
-    newTag: c9f05be85b110de65b84fa51017ff03601509652
+    newTag: 4163d56f783218201b65284d5dbe12531a4339fb
```

Comment:
Just to make sure this is not an issue on our end, I confirmed that `quay.io/konflux-ci/build-service:4163d56f783218201b65284d5dbe12531a4339fb` does not exist (probably release pipeline failure for this image).